containerd-shim-wws-v1/src/main.rs

```diff
@@ -0,0 +1,221 @@
+use chrono::{DateTime, Utc};
+use containerd_shim as shim;
+use containerd_shim_wasm::sandbox::error::Error;
+use containerd_shim_wasm::sandbox::instance::{EngineGetter, InstanceConfig};
+use containerd_shim_wasm::sandbox::oci;
+use containerd_shim_wasm::sandbox::{Instance, ShimCli};
+use log::{error, info};
+use std::path::Path;
+use std::path::PathBuf;
+use std::sync::mpsc::channel;
+use std::sync::mpsc::Sender;
+use std::sync::Arc;
+use std::sync::{Condvar, Mutex};
+use std::thread;
+use tokio::runtime::Runtime;
+use wws_config::Config;
+use wws_router::Routes;
+use wws_server::serve;
+
+/// URL to listen to in wws
+const WWS_ADDR: &str = "0.0.0.0";
+const WWS_PORT: u16 = 80;
```

Comment:
What do you think about using 3000 or something above the privileged ports range?

Reply:
That's totally fine! I will update it to use the `3000` port.